doc/configuration.md

```diff
@@ -0,0 +1,206 @@
+## Configuration
+
+As of Ziggurat version 3.13.0, all the official Kafka configs Kafka configurations for [Streams API](https://docs.confluent.io/platform/current/installation/configuration/streams-configs.html), [Consumer API](https://docs.confluent.io/platform/current/installation/configuration/consumer-configs.html) and [Producer API](https://docs.confluent.io/platform/current/installation/configuration/producer-configs.html) are supported.
+
+All Ziggurat configs should be in your `clonfig` `config.edn` under the `:ziggurat` key.
+
+## Table of Contents
+
+1. [General Configurations](#general-configurations)
+2. [Stream Router Configurations](#stream-router-configurations)
+    - [Channels](#channels)
+    - [Producer](#producer)
+3. [Batch Routes](#batch-routes)
+4. [SSL](#ssl)
+5. [StatsD](#statsd)
+6. [Sentry](#sentry)
```

Comment:
add concepts of prometheus exporter here which is not present

Reply:
added